src/main/java/com/internet/shop/dao/OrderDao.java

```diff
@@ -1,4 +1,19 @@
 package com.internet.shop.dao;
 
+import com.internet.shop.model.Order;
+import java.util.List;
+import java.util.Optional;
+
 public interface OrderDao {
+    public Order create(Order order);
```

Comment:
```suggestion
    Order create(Order order);
```
public access modifier is default modifier in interface, so it's redundant here, same in other methods in interface